charmcraft/application/commands/store.py

```diff
@@ -1992,7 +1992,7 @@ def run(self, parsed_args: argparse.Namespace) -> int:
             if const.STORE_REGISTRY_ENV_VAR in os.environ:
                 # If the user has specified a registry to use, replace what the store
                 # gives with that registry.
-                registry_url = os.environ[const.STORE_REGISTRY_ENV_VAR][7:]
+                registry_url = os.environ[const.STORE_REGISTRY_ENV_VAR][8:]
```

Comment:
This looks fragile.  Maybe add a comment explaining what the list looks like that we're slicing here, or why the first n elements are skipped, etc.

Reply:
Good point. I've made this more obvious.

Reply:
Much better, thanks!